Terraform/GCP/k8s/elastic.tf

```diff
@@ -8,14 +8,19 @@ resource "kubectl_manifest" "elastic_ssl" {
   spec:
     dnsNames:
       - elasticsearch-master
-      - elasticsearch-master.outofschool.svc
-      - elasticsearch-master.outofschool.svc.cluster.local
+      - elasticsearch-master-0
```

Comment:
maybe still add FQDNs, it does not cost anything but may be needed somewhere later

Reply:
ok 
namespace is default
FQDNs must be other
elasticsearch-master.default.svc
elasticsearch-master.default.svc.cluster.local

Reply:
will we have namesapce outofschool ? I don't see now
will elasticsearch-master.outofschool.svc add?